x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/results/MlChunkedTextExpansionResults.java

```diff
@@ -21,7 +21,7 @@
 import java.util.Objects;
 import java.util.stream.Collectors;
 
-public class InferenceChunkedTextExpansionResults extends ChunkedNlpInferenceResults {
+public class MlChunkedTextExpansionResults extends ChunkedNlpInferenceResults {
```

Comment:
Renamed following the convention that the inference plugin classes start with `Inference` and this is an ml class